frontend/packages/console-app/src/components/console-operator/ConsolePluginEnabledStatusDetail.tsx

```diff
@@ -28,12 +27,12 @@ const ConsolePluginEnabledStatusDetail: React.FC<DetailsItemComponentProps> = ({
     consoleOperatorConfig?.spec?.plugins,
   ]);
 
-  return consoleOperatorConfigLoaded ? (
+  return consoleOperatorConfigLoaded && pluginName ? (
```

Comment:
All k8s objects should have a name, do we really need to check up on `pluginName` explicitly?

Reply:
We have null checks for `pluginName`, so in theory this could be null. This just prevents some linting errors